### ISSUE TITLE
Consider differences between "Pedagogy" and "Andragogy" 

### ISSUE THREAD

**Issue by PeteEmbleton** [carpentries/instructor-training#1229]
It can be really useful to consider the differences between way children and adults learn 
https://elmlearning.com/pedagogy-vs-andragogy/#:~:text=At%20a%20glance%2C%20andragogy%20refers,evaluation%2C%20and%20acquisition%20of%20knowledge

It is important to understand and consider this, particularly if we do want to look at using this sort of content with younger people. I teach both young people and adults and find that the same content may get 80% success so i do need to change my approach.

It could be good to have a call out in the instructor training to get people to think about the differences

**Comment by mariakna** [comment]
A very good point Peter. 

I also think that it is equally important to consider differences between teaching to younger (under 35) vs older adults (45+). There seems to be a consensus in the scientific community that many of the age-related memory difficulties are rooted in retrieval. For example, it has been suggested that, while crystalised intelligence remains intact with aging, access to and control of a person's knowledge diminishes with age (e.g., Craik & Bialystok, 2006). This is not trivial as it implies differences in information retrieval in younger vs. older adults. And what if we are talking about information that has been only recently learned? This adds more complexity to the issue.

It is also widely known that older adults have reduced working memory capacity (e.g., Cabeza, Anderson, Locantore, & McIntosh, 2002; Long & Shaw, 2000; this also applies to kids by the way). Moreover, there are profound changes to quantity and quality of sleep (e.g., sleep spindle reduction, reduced SWA, frequent awakenings during REM, etc), and sleep is an important factor in memory consolidation. There is also the hyper-binding theory (Campbell et al., 2012) that posits that older adults encode more distracting information than younger adults due to a lessened ability to inhibit irrelevant information. Another important thing is that there is much more interindividual variability across older than younger adults so it's much harder to predict a learning outcome and the difficulties on the way if we are dealing with older adults.

I could go on and on (and the things I mentioned also partly apply to the children vs adults dichotomy), but the point is that, while we know quite a bit about how memory processes change across development, I feel like not much of this knowledge is applied in classrooms (especially in the more traditional ones). But these differences are not trivial, and making use of this information would greatly enhance the learning experience for the instructors and the learners alike. 

And on a different but a related note: the learning context matters too! To give an example, in 2nd language teaching (and in aphasia therapy) words are always taught in semantically related sets (e.g., all animals together, or food, or pieces of furniture, etc), although there is no empirical evidence that this approach is beneficial for learning. Instead, it is better to group items by themes, e.g., learning the names of the concepts that have to do with funerals (tears, grief, handkerchief, flowers etc). I would be interested in thinking about how this translates to teaching computational languages - I have some thoughts but it would be cool to discuss this with the community!

Haha that's a very long post about so many things at once :) But I got inspired by your message.

**Comment by karenword** [comment]
Thank you both @PeteEmbleton and @mariakna for your helpful contributions!

It is true that "andragogy" is a more accurate term for what we teach, and I like the idea of a callout on this. The primary reason we have maintained "pedagogy" as our term of reference is because this is a more commonly recognized term, and relying upon it as an imperfect reference allows us to minimize introduction of new jargon. However, a callout would be an appropriate way to recognize this inaccuracy without asking people to apply a new term.

I am more hesitant to address age-related memory characteristics in the context of this curriculum. Variation at this level is very unlikely to impact how we teach short-format workshops of this kind, in part because factual memory is likely to be very limited for all participants. This is why we focus on building mental models and supporting motivation to persevere beyond the workshop. Beyond the question of relevance, though, treatment of these sorts of cognitive distinctions between people -- whether related to age or other characteristics -- can get very problematic very quickly. Hosting a nuanced conversation on such a subject really demands deep expertise and is not something that can easily be conveyed in an open curriculum of this kind.

**Comment by brownsarahm** [comment]
yes thank you @PeteEmbleton and @mariakna for this discussion. I learned something new here. 

Our curriculum is already overflowing with content and trainees regularly report that the amount of supplemental material is overwhelming.  We’re currently pretty narrowly focused on updates that refine and narrow the curriculum rather than adding new content. We've decided not to add this callout in favor of not increasing the cognitive load on trainees, as we try to focus them in on specific practices and the Carpentries mission during Instructor Training.  


Broadly, however, as a community of practice, we are interested in supporting this type of discussion, so we’re going to move this issue to a different location for future reference and continued discussion.
